packages/react-tinacms-github/src/toolbar-plugins/pull-request/PRModal.tsx

```diff
@@ -93,6 +93,19 @@ export const PRModal = () => {
     baseBranch = `${baseOwner}:${baseBranch}`
   }
 
+  if (workingBranch === baseBranch) {
```

Comment:
Should we also be checking if the Base Repo is the same as the working repo? 
We will want this to still work with forks

Reply:
Nope, the baseBranch and workingBranch variables contain the owner name if they're different.

Reply:
Confusing name, I might change them 🤔